src/main/scala/com/microsoft/kusto/spark/utils/KeyVaultADALAuthenticator.scala

```diff
@@ -0,0 +1,70 @@
+package com.microsoft.kusto.spark.utils
+
+import java.net.MalformedURLException
+import java.util.concurrent.{ExecutionException, ExecutorService, Executors, Future}
+
+import com.microsoft.aad.adal4j.{AuthenticationContext, AuthenticationResult, ClientCredential}
+import com.microsoft.azure.keyvault.KeyVaultClient
+import com.microsoft.azure.keyvault.authentication.KeyVaultCredentials
+import com.microsoft.rest.credentials.ServiceClientCredentials
+
+
+/**
```

Comment:
I don't think we used this format elsewhere in the connector. Am I right? 
I don't object the format, please just verify we're aligned across files